auth/range_perm_cache.go

```diff
@@ -163,24 +163,35 @@ func checkKeyPerm(cachedPerms *unifiedRangePermissions, key, rangeEnd []byte, pe
 
 func (as *authStore) isRangeOpPermitted(tx backend.BatchTx, userName string, key, rangeEnd []byte, permtyp authpb.Permission_Type) bool {
 	// assumption: tx is Lock()ed
```

Comment:
`be.BatchTx()` is already locked-- is there a path where `rangePermCacheMu` is accessed without holding the tx lock?